reference/api/keys.mdx

```diff
@@ -111,6 +111,10 @@ For security reasons, we do not recommend creating keys that can perform all act
 | **`network.get`**      | Provides access to the [get the network object](/reference/api/network#get-the-network-object) endpoint                                                                                                                                                                                                                                                                |
 | **`network.update`**   | Provides access to the [update the network object](/reference/api/network#update-the-network-object) endpoint                                                                                                                                                                                                                                                                  |
 | **`chatCompletions`**  | Provides access to the [chat completions endpoints](/reference/api/chats). Requires experimental feature to be enabled                                                                                                                                                                                                                   |
+| **`webhooks.get`**     | Provides access to the [get webhooks](/reference/api/webhooks#get-all-webhooks) endpoints                                                                                                                                                                                                                                                                |
+| **`webhooks.create`**  | Provides access to the [create webhooks](/reference/api/webhooks#create-a-webhook) endpoint                                                                                                                                                                                                                                                                  |
+| **`webhooks.update`**  | Provides access to the [update webhooks](/reference/api/webhooks#update-a-webhook) endpoint                                                                                                                                                                                                                                                                  |
+| **`webhooks.delete`**  | Provides access to the [delete webhooks](/reference/api/webhooks#delete-a-webhook) endpoint                                                                                                                                                                                                                                                                  |
 
```

Comment:
Maybe it was missing from the PRD but we also have `webhooks.*` which includes all 4